.github/workflows/build.yml

```diff
@@ -0,0 +1,78 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+      - /^\d+\.\d+\.\d(-alpha\.\d+|-beta\.\d+|-rc\.\d+)?$/
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Test, Build & Release
+    runs-on: "ubuntu-latest"
+    env:
+      GH_ACTIONS_NODE_VERSION: "10"
+      LOGS_DIR: /tmp/code-style/logs
+      LOGS_FILE: /tmp/code-style/logs/build-perf.log
+      TZ: "Europe/Brussels"
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 10
+        uses: actions/setup-node@v1
```

Comment:
Can you add a comment above this line about where we can find info over `actions/setup-node@v1`?